packages/react-magma-dom/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -426,3 +430,64 @@ WithDropdown.args = {
   ...Default.args,
 };
 WithDropdown.parameters = { ...Default.parameters };
+
+const StyledTabsContainer = styled(TabsContainer)`
+  [data-testid='tabsWrapper'] {
+    overflow-x: visible;
+  }
+`;
+
+const StyledButton = styled(Button)`
+  margin-inline: 10px;
+`;
+
+const K12Button = styled.button`
```

Comment:
I don't want us to use K12 as a naming convention within Magma examples (storybook or docs), so lets just call it `CustomButton` or anything else that's generic.